.github/workflows/e2e-docker-server-111.yml

```diff
@@ -35,25 +30,39 @@ on:
       send_slack_notification:
         description: 'Send Slack notification?'
         required: false
-        default: false
+        default: true
         type: boolean
       run_ha_test:
         description: 'Run HA tests or not'
         required: false
         default: true
         type: boolean
-      upload_logs:
-        description: 'Upload logs to AWS'
+      test_case_to_run:
+        description: 'Specific test case to run (leave empty to run all)'
         required: false
-        default: false
-        type: boolean
+        default: 'TestSingleNodeFailure'
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: false
 
 jobs:
+
+  deploy:
+    uses: ./.github/workflows/bare-metal-deploy.yml
+    with:
+      runs_on: self-hosted
+      cluster: c141
```

Comment:
Using a variable here as we do for the integration tests would be more flexible. We could also remove cluster identifier from the filename, currently this doesn't match up anyway.

Reply:
fixed in [6b909bc](https://github.com/simplyblock-io/sbcli/pull/445/commits/6b909bc7490f5678a35d5f714c4fadbde788598e)